### ISSUE TITLE
support "follow the traffic" visual approaches

### ISSUE THREAD

**Issue by mmp** [mmp/vice#60]
It would be nice to have these as well. Following the potential syntax in #59, maybe `T/2/5` for "traffic at your 2 o'clock, 5 miles, report in sight"? Then we need a "follow the traffic" command, `FT`?, then as in #59 something like `CV31R` for the approach clearance.

**Comment by checkandmate1** [comment]
I think `T 2 5` would be better just because `/` is sometimes an awkward key to press

**Comment by mmp** [comment]
True, though we generally use spaces to separate commands when giving more than one to an aircraft. It is a little annoying to have to give o'clock and distance; maybe it should just be `TR` or `TF` to look for traffic and not have the user have to include that other stuff in the command. (`TR` is not great since we have `TR123` for "turn right heading 123"...)

**Comment by mmp** [comment]
Added in efb895fa1245756e13ffc0f2d99330735c3de922
